developer/pages-router/localization.mdx

```diff
@@ -6,105 +6,114 @@ description: "This guide walks you through setting up localization for your Make
+To match the locales on the screenshot above, update `next.config.ts` to be like this:
 
 <NextConfigLocalizedExample />
 
-Note that in `next.config.mjs`, you also need to put your `defaultLocale` in `locales`.
-For example, here, we're adding `en-US` to the `locales`.
-
-It is important to match the locales and the default locale on the site settings to the locales configured in `next.config.mjs`. Otherwise
-the pages on the builder will not load properly.
+Note that in `next.config.ts`, you also need to put your `defaultLocale` in `locales`.
+In this example, that would be `en-US`.
 
-</Step>
+## Fetch snapshots by locale
 
-<Step title="Update snapshot fetching">
-Once you set up locales in the site settings and in the Next.js config, you need to pass the locale from `getStaticProps` to `getPageSnapshot`.
+Update your catch-all route to fetch the page snapshots by locale. This requires you to pass the locale from `getStaticProps` to `getPageSnapshot`. This code is usually located in `[[...path]].tsx`, but it might be different depending on your setup.
 
 <PagesCatchAllLocalizedExample />
 
-This code is usually located in `[[...path]].tsx`, but it might be different depending on your setup.
+You'll also need to pass the locale to the `<ReactRuntimeProvider>` component by updating your `_app.tsx` like so.
 
-</Step>
+<PagesAppLocalized />
 
-<Step title="Edit your pages in the builder">
+## Edit your localized pages
 
-Once you've set everything up, you should be able to switch to the locale using the locale switcher on the builder.
+Once you've set everything up, you should be able to switch to the locale using the locale switcher in the Visual Builder.
 
 <Frame>
-  ![Manage locales in settings](/images/localization-locales-switcher.png)
+  ![Locale switcher in the Visual
+  Builder](/images/developer/guides/localization/locale-switcher.jpeg)
 </Frame>
 
-<Note>
-
-You can customize the path for each page in each locale. For example, if you have a company page at `example.com/company`, you can create the Spanish version of the page at `example.com/es/compania` or `example.es/compania`.
+The first time you switch to a new locale (in this case switching from `en-US` to `es-ES`), you'll notice a message that informs you that the localized page inherits from the default locale. To stop inheriting from the default locale and start customizing the content for the localized page, click the **Edit for this locale** button.
 
-</Note>
+<Frame>
+  ![Option in the Visual Builder to edit a localized
+  page.](/images/developer/guides/localization/edit-locale-message-2.jpeg)
+</Frame>
 
-</Step>
+Note that you will continue to see the message about inheriting from the base locale until you click that button. **This also means that until you decide to edit this localized page, it will continue to serve the page for the default locale.**
```

Comment:
I'd strike the first sentence here since it's stating something that's expected (why wouldn't they continue seeing the message if they haven't taken action?), and say something like

> Note that your site will keep serving the default locale’s version of the page until you edit the localized page and publish your changes.

Reply:
The main reason I suggested to remove that sentence is that it IMHO detracts from the much more important part of the note, which is "your site will keep serving the default locale’s version of the page until you edit the localized page and publish your changes." 

I see that we are bolding the second sentence (which still needs to be updated from "until you decide to edit" to "until you edit and publish your changes"), but altogether the phrasing and the placement of the more important sentence after the less important one feels like a crutch to me. 

This is arguably the main thing about localized pages and fallbacks we want the user to understand, and I think it should be presented on its own.

Reply:
Updated. Let me know if that works!

```
<Frame>
  ![Option in the Visual Builder to edit a localized
  page.](/images/developer/guides/localization/edit-locale-message-2.jpeg)
</Frame>

**Note that until you decide to edit and publish this localized page, it will continue to serve the page for the default locale.** After choosing to edit the localized page, content changes are only associated with this locale and do not affect the base locale.
```

Reply:
How about this:

**Note that until you edit and publish the localized page, it will continue displaying content from the default locale**. Any changes you make after clicking **Edit for this locale** will apply only to the localized version and won’t affect the base locale.

?

Reply:
Updated with your wording!